python/doc/theory/data_analysis/gaussian_calibration.rst

```diff
@@ -6,6 +6,16 @@ Gaussian calibration
 Introduction
 ~~~~~~~~~~~~
 
+In this page, we present the method used in the :class:`~openturns.GaussianLinearCalibration`
+and :class:`~openturns.GaussianNonLinearCalibration` classes.
+The Gaussian calibration is a particular case of Bayesian calibration
```

Comment:
Link to a previous suggestion which has been buried by GitHub: https://github.com/openturns/openturns/pull/2060#discussion_r1165915860

Reply:
Ok. This must be fixed now.

Reply:
Indeed it is. I will probably review the PR one last time after the CI manages to compile the documentation.

Reply:
Ok. So I resolve this conversation.